/* \file mailroom.h
 * \description provides the mailroom node
 * \author Jesse Joseph
 */

#include <stdio.h>
#include <string>
#include "ros/ros.h"
#include "std_msgs/String.h"
#include "mailroom/ATSCsignal.h"
#include "mailroom/drone_cmd.h"
#include "mailroom/drone_status.h"
#include "mailroom/drone_telemetry.h"

int main(int argc, char **argv);

